pkg/atlas/meta.go

```diff
@@ -1,10 +1,20 @@
 package atlas
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"net/http"
 )
 
+// PathToProviderJSON is the path to the json file
+var PathToProviderJSON = "providers.json"
+
+// ProvidersFile represents all of the providers that have been set in the provider.json file
+type ProvidersFile struct {
```

Comment:
[nit] may be `ProvidersConfig` or whatever? The `File` suffix doesn't seem very clear to me

Reply:
Agreed.